client/src/interface/BookingRequest.ts

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+import {User} from './User'
+
+export interface BookingRequest {
+  _id: string;
+  user_id: User;
+  sitter_id: string;
+  start_date: any;
+  end_date: any;
+  accepted?: boolean;
+  declined?: boolean;
+  paid?: boolean;

```

Comment:
Is there ever an instance where this is potentially null? Booleans are only ever true or false.

Reply:
If this is a concern of when we are sending the request, there could be a separate request object that accepts just the required data and then the response would match the BookingRequest interface

Reply:
I believe it is fine without the question mark -- I've removed them.